nni/experiment/experiment.py

```diff
@@ -169,7 +169,7 @@ def run(self, port: int = 8080, debug: bool = False) -> bool:
             while True:
                 time.sleep(10)
                 status = self.get_status()
-                if status == 'STOPPED':
+                if status == 'DONE' or status == 'STOPPED':
```

Comment:
Why add `DONE`? What is the behavior of `SUCCEED` or `FAILED`?

Reply:
It is the status of experiment, not trials. And we expect this process automatic exit when the experiment is finished. You reminded me maybe we should also add `ERROR`?

Reply:
Seems `ERROR` already exist in line 174.

Reply:
Oh yes, I miss it.